api/web/base.py

```diff
@@ -208,6 +210,35 @@ def validate_oauth_token(self, access_token, timestamp):
 
         return uid
 
+
+    @log_access(AccessType.user_login)
+    def log_in(self):
+        """
+        Return succcess boolean if user successfully authenticates.
+
+        Used for access logging.
+        Not required to use system as logged in user.
+        """
+
+        if not self.uid:
+            self.abort(400, 'Only users may log in.')
+
+        return {'success': True}
```

Comment:
Exactly. I mean it will also give them a way to check that their method of auth works without having to make a dummy call to /sessions or something.